### PR TITLE
Feature/test mailinglist

### PR DESCRIPTION
**Pull request by maksat990rd**

### CODE REVIEW

File: src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Test
+    @Order(1)
+    @Rollback(value = false)
+    void saveMailingList() {
+        MailingList mailingList = new MailingList(1L,"test","test","test",
+                LocalDateTime.now());
+        Assertions.assertThat(mailingList.getId()).isGreaterThan(0);
+    }
+    @Test
```
Comment (i-Coderr) [inline src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java:30]: Please write to the next line

File: src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        MailingList mailingList = repository.findById(1L).orElse(
+                new MailingList(1L,"test","test","test", LocalDateTime.now()));
+        Assertions.assertThat(mailingList.getId()).isEqualTo(1L);
+    }
+
+    @Test
+    @Order(3)
+    void getAllMailingLists() {
+        List<MailingList> mailingLists = repository.findAll();
+        Assertions.assertThat(mailingLists.size()).isEqualTo(0);
```
Comment (i-Coderr) [inline src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java:46]: ???

**Review by i-Coderr — COMMENTED** [review]
Commented at 12.24 pm

File: src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java (hunk 2 above)
Comment (i-Coderr) [inline src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java]: ???

File: src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java (hunk 1 above)
Comment (maksat990rd) [reply src/test/java/kg/peaksoft/giftlistb6/db/services/MailingListServiceTest.java:30]: ok

**Review by i-Coderr — APPROVED** [review]
Ok

**Pull request merged**